src/main/java/pl/javastart/task/Chair.java

```diff
@@ -0,0 +1,14 @@
+package pl.javastart.task;
+
+public class Chair {
+    boolean canRotate;
+    double heightMin;
+    double heightMax;
+    String producer;
+    Chair(String prod, double heiMax, double heiMin, boolean rotate){
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj. W instrukcji jest też informacja jak ustawić dla wszystkich nowych projektów. https://bootcamp.javastart.pl/strona/jakosc-kodu